src/Symfony/Component/Serializer/Normalizer/ConstraintViolationListNormalizer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\Validator\ConstraintViolationListInterface;
+
+/**
+ * A normalizer that normalize a ConstraintViolationListInterface instance.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class ConstraintViolationListNormalizer implements NormalizerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function normalize($object, $format = null, array $context = array())
+    {
+        $violations = array();
+        $messages = array();
+        foreach ($object as $violation) {
+            $violations[] = array(
+                'propertyPath' => $violation->getPropertyPath(),
+                'message' => $violation->getMessage(),
```

Comment:
I would also add the code (`$violation->getCode()`) for the machine-readable stable identifier of the violation (the message is not stable as it is translated)